tests/Ev.ServiceBus.AsyncApi.UnitTests/Ev.ServiceBus.AsyncApi.UnitTests.csproj

```diff
@@ -1,13 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
     
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
-
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
That being said, the unit test can and should have multipl target frameworks ^^
they will get executed in all of the specified frameworks.

so for all the test they should have net6.0 and net8.0